eruditus/app_commands/ctf/__init__.py

```diff
@@ -1347,6 +1349,100 @@ async def submit(
+        Show the list of solvers for the provided/current challenge.
+        """
+        async def followup(content: str, ephemeral=True, **kwargs) -> None:
+            if not interaction:
+                return
+            await interaction.followup.send(content, ephemeral=ephemeral, **kwargs)
+        
+        await interaction.response.defer()
+
+        if name is None:
+            challenge = get_challenge_info(thread=interaction.channel_id)
+            if challenge is None:
+                await followup(
+                    (
+                        "Run this command from within a challenge thread, "
+                        "or provide the name of the challenge you wish to stop "
+                        "working on."
+                    )
+                )
+                return
+        else:
+            challenge = get_challenge_info(name=name)
+            if challenge is None:
+                await followup(
+                    "No such challenge."
+                )
+                return
+        
+        ctf = get_ctf_info(guild_category=interaction.channel.category_id)
+        ctx: PlatformCTX = PlatformCTX.from_credentials(ctf["credentials"])
+        platform = await match_platform(ctx)
+        
+        if not platform:
+            await followup("Invalid URL set for this CTF, or platform isn't supported.")
+            return
+
+        solvers = []
+        
+        async for solver in platform.pull_challenge_solvers(
+            ctx=ctx,challenge_id=challenge["id"], limit=10
+        ):
+            solvers.append(solver)
+
+        num_solvers = len(solvers)        
+        if(num_solvers==0):
+            await followup("This challenge has not been blooded yet.")
+            return
+        
+        me = await platform.get_me(ctx)
+        our_team_name: str = me.name if me is not None else TEAM_NAME
+
+        embed = None
+        num_fields = 0
+        
+        for idx, solver in enumerate(solvers,start=1):
+            # If we reached Discord's maximum number of fields per
+            # embed, we send the previous one and create a new one.
+            if num_fields % 25 == 0:
+                if num_fields != 0:
+                    await interaction.followup.send(embed=embed)
+                    embed = None
+
+                if embed is None:
+                    embed = discord.Embed(
+                        title=f"{challenge['name']} - {num_solvers} solvers",
+                        colour=discord.Colour.green(),
+                    )
+            
+            if (idx==1):
+                name = f"🩸 - {solver.team.name}"
+            elif(solver.team.name==our_team_name):
+                name = f"🔴 - {solver.team.name}"
+            else:
+                name = f"{idx} - {solver.team.name}"
+            
+            embed.add_field(
+                name=name,
+                value=f"{time_since(solver.solved_at)}",
+                inline=False,
+            )
```

Comment:
Instead of introducing a new helper function for this, it would be cleaner and more efficient to use Discord's built-in relative timestamp format (<t:${timestamp}:R>). It automatically shows the relative time and updates in real time, so there's no need to handle that manually.